packages/core/src/app/order/OrderSummarySubtotals.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { mapFromPayments } from '../giftCertificate';
+
+import { getOrder } from './orders.mock';
+import OrderSummarySubtotals from './OrderSummarySubtotals';
+
+describe('OrderSummarySubtotals', () => {
+    it('renders component', () => {
+        const order = getOrder();
+
+        render(
+            <OrderSummarySubtotals
+                coupons={order.coupons}
+                discountAmount={order.discountAmount}
+                giftCertificates={order.payments && mapFromPayments(order.payments)}
+                handlingAmount={order.handlingCostTotal}
+                shippingAmount={order.shippingCostTotal}
+                subtotalAmount={order.orderAmount}
+                taxes={order.taxes}
+            />,
+        );
+
+        expect(screen.getByText('Tax')).toBeInTheDocument();
+        expect(screen.getByText('20% off each item')).toBeInTheDocument();
+        expect(screen.getByText('savebig2015')).toBeInTheDocument();
+        expect(screen.getByText('$5.00 off the shipping total')).toBeInTheDocument();
+        expect(screen.getByText('279F507D817E3E7')).toBeInTheDocument();
+        expect(screen.getByTestId('cart-subtotal')).toBeInTheDocument();
+        expect(screen.getByTestId('cart-shipping')).toBeInTheDocument();
+        expect(screen.getByTestId('cart-handling')).toBeInTheDocument();
+        expect(screen.getByTestId('cart-taxes')).toBeInTheDocument();
+        expect(screen.getByTestId('cart-discount')).toBeInTheDocument();
+        expect(screen.getByTestId('cart-gift-certificate')).toBeInTheDocument();
+        expect(screen.getAllByTestId('cart-coupon')).toHaveLength(2);
+        expect(screen.queryByTestId('cart-gift-wrapping')).not.toBeInTheDocument();
+        expect(screen.queryByTestId('cart-store-credit')).not.toBeInTheDocument();
+    });
+
+    describe('Tax summary is not rendered if tax is inclusive', () => {
+        it('Renders component without tax summary', () => {
```

Comment:
```suggestion
        it('renders component without tax summary', () => {
```

Use lower case as the beginning of the sentence is `it`. 🍹